docs/articles/documentation/extending-testcafe/reporter-plugin/reporter-methods.md

```diff
@@ -89,13 +89,14 @@ Property         | Type             | Description
 `durationMs`     | Number           | The time spent on test execution (in milliseconds).
 `unstable`       | Boolean          | Specifies if the test has been marked as unstable.
 `screenshotPath` | String           | The directory path where screenshots have been saved to.
+`skipped`        | Boolean          | Specifies if the test was skipped.
 
 **Example**
 
 ```js
 reportTestDone (name, testRunInfo) {
     const hasErr = !!testRunInfo.errs.length;
-    const result = hasErr ? `passed` : `failed`;
+    const result = testRunInfo.skipped ? 'skipped' : hasErr ? `passed` : `failed`;
```

Comment:
let's stay as it here. But for more difficult cases please use condition in ternary)